code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -30,14 +30,17 @@
 	ammo_x_offset = 3
 
 /obj/item/gun/energy/laser/captain
-	name = "antique laser gun"
+	name = "commemorative laser gun"
```

Comment:
I personally think the name should remain unchanged

Reply:
An antique gun would be old, outdated and completely inferior to modern weapons. The captain's gun is not an antique at all unless Nanotrasen has failed to produce new technology for a very long time - it outshines everything in the armory at the start of each shift.

Reply:
If its changing the traitor obj needs to be done too

Reply:
Ah whoops.